.circleci/config.yml

```diff
@@ -28,24 +28,38 @@ jobs:
   build:
     machine:
       image: ubuntu-1604:201903-01 
-    working_directory: ~/inbox
+    working_directory: ~/sync-engine
 
     steps:
       - checkout
 
       - run:
-          name: build nylas image
-          command: docker-compose build nylas
+          name: build sync-engine image
+          command: docker-compose build app
 
       - run:
           name: run tests
           command: |
             docker login -u $DOCKER_USER -p $DOCKER_PASS
-            docker-compose run nylas bash -ec '
+            docker-compose run app bash -ec '
               pip install -e . \
               && bin/wait-for-it.sh mysql:3306 \
               && NYLAS_ENV=test py.test tests/
             '
+      - run:
+          name: Push the tested image to docker hub
+          environment:
+            PROJECT: sync-engine
+            PROJECT_NAMESPACE: closeio
+          command: |
+            PROJECT_VSN="$CIRCLE_BRANCH"
+            PROJECT_SHA="$CIRCLE_SHA1"
+            docker login -u $DOCKER_USER -p $DOCKER_PASS
+            # tag and push both sync-engine:master and sync-engine:<sha1>
+            docker tag "${PROJECT}_app" "${PROJECT_NAMESPACE}/${PROJECT}:branch-${PROJECT_VSN}"
```

Comment:
where are `PROJECT` and `PROJECT_NAMESPACE` set?

Reply:
@AlecRosenbaum not sure if I understand correctly but it's set under `enviornment:` couple of lines above.

Reply:
🤦‍♂️ Yep, my bad.